### ISSUE TITLE
Can't use filter after mac update to latest OS. 

### ISSUE THREAD

**Issue by patrickcash1** [sequelpro/sequelpro#3855]
You can't see the filter options now once I updated to latest mac OS. You click cmd+f but you can't see the input box. 
![image](https://user-images.githubusercontent.com/3642399/197849841-0ec7afd7-e266-47d6-b0f3-86f32e493103.png)

**Comment by JackSzwergold** [comment]
Just confirming that is the case with me as well on Sequel Pro [Nightly Build 5446 (97c1b857)] running on macOS Ventura 13.0 (22A380) running on a 2020 Intel MacBook Air.

If I hit <kbd>Command</kbd>+<kbd>F</kbd> I can see the small filter line at the top of the columns area nudge down, I see  a wee bit of blue highlight and that’s it. I can definitely filter on whatever the first column is, but past that? Nothing because it’s all hidden away.

<img width="855" alt="Screenshot 2022-10-25 at 2 39 43 PM" src="https://user-images.githubusercontent.com/745105/197855178-8213b335-df81-4a59-be88-4a9edc5bb69f.png">

**Comment by mirozahorak** [comment]
Ventura MBP M1 16inch - having the same issue

**Comment by JackSzwergold** [comment]
Active development on Sequel Pro is effectively dead. 

Seems like the solution to this issue is to move over to Sequel Ace as [explained here](https://github.com/sequelpro/sequelpro/issues/3705). Active development on Sequel Pro is effectively dead. Sequel Ace can be found [here](https://github.com/Sequel-Ace/Sequel-Ace) and it has instructions on how to install it via the Apple App Store or even Homebrew.

**Comment by acegilz** [comment]
I had the same issue with the nightly build. I have been using it for 2 years because of another issue I had with the main release.. 

Now I installed the latest stable release (1.1.2) and filtering works again, hopefully I won't see the previous issue again that I don't even recall what it was.

**Comment by JackSzwergold** [comment]
Just confirming what @acegilz just posted: I downloaded Sequel Pro 1.1.2 [Build 4541 (60211a)] and filtering works fine on macOS 13 (Ventura). 

<img width="817" alt="Screenshot 2022-10-26 at 1 05 02 PM" src="https://user-images.githubusercontent.com/745105/198091422-afac7d81-4b88-4a8c-a4c1-49fe468cf036.png">

FWIW, everyone was using the nightly build because in 2019 Sequel Pro was crashing on macOS 10.14 (Mojave) and the nightly build fixed that issue [as explained in this other issue comment](https://github.com/sequelpro/sequelpro/issues/3360#issuecomment-500719797).

**Comment by kotygomez** [comment]
For those ones who still care, I was having the same issues and I'm finally able to bring the filter box back. I'm on Mac OS Ventura (13.0) and Sequel Pro Night build 5446. 

So, if your filter fields are gone, a way to bring them back is:

1. First make sure the "show/hide table filter" icon at the bottom is selected. This will show the "Filter" button at the top but the fields are hidden, just like on the screenshot where the problem was initially reported.

2. Open a new connection tab (not a new window), in my case the bar with the 'Add Filter' button at the top shows up.
<img width="1175" alt="image" src="https://user-images.githubusercontent.com/4741730/201805813-cc16b3e3-de1b-4633-bb8f-c50800a67b2f.png">

3. Now, you should be able to select a table and use the filters by clicking on the 'Add Filter' button. 

I hope this helps!

PS: I think the filters go missing (get hidden) when switching back and forth from the query window, depending on which state the "show/hide table filter" is in.

**Comment by treyseals** [comment]
Having the same issue as described. Is SequelPro dead?

**Comment by JackSzwergold** [comment]
@treyseals Yes, Sequel Pro is — for all intents and purposes — dead and no lingering being actively maintained. I said as much in [this other comment](https://github.com/sequelpro/sequelpro/issues/3855#issuecomment-1292091447).

**Comment by lixiaohui8636** [comment]
I found a way. Select the filter button at the bottom,

1. Make sure it is selected
![image](https://user-images.githubusercontent.com/19178735/203464338-c3a3ca2f-7e01-4b22-89b7-c4d12482d355.png)
2. Exit the software
3. Restart the software. At this time, the filter button at the bottom is selected. The Add Filter displayed above can be clicked at this time, or the command+F can be directly used to display the search box
![image](https://user-images.githubusercontent.com/19178735/203464411-3b3fb289-ca8e-48fb-836a-2842075affbe.png)

Command+F is shown in the figure below

![image](https://user-images.githubusercontent.com/19178735/203464464-8b7e78e3-4581-4d24-ab80-12854183ade3.png)


I'm on Mac OS Ventura (13.0) and Sequel Pro Night build 5446.

**Comment by JackSzwergold** [comment]
@lixiaohui8636 Wow! This works great! So the key is always having filtering enabled via that bottom menu. Great work discovering this!

**Comment by abmn614** [comment]
> I found a way. Select the filter button at the bottom,
> 
> 1. Make sure it is selected
>    ![image](https://user-images.githubusercontent.com/19178735/203464338-c3a3ca2f-7e01-4b22-89b7-c4d12482d355.png)
> 2. Exit the software
> 3. Restart the software. At this time, the filter button at the bottom is selected. The Add Filter displayed above can be clicked at this time, or the command+F can be directly used to display the search box
>    ![image](https://user-images.githubusercontent.com/19178735/203464411-3b3fb289-ca8e-48fb-836a-2842075affbe.png)
> 
> Command+F is shown in the figure below
> 
> ![image](https://user-images.githubusercontent.com/19178735/203464464-8b7e78e3-4581-4d24-ab80-12854183ade3.png)
> 
> I'm on Mac OS Ventura (13.0) and Sequel Pro Night build 5446.

Great work!

**Comment by sleemkeen** [comment]
@lixiaohui8636  solution Worked

**Comment by slasho** [comment]
> I found a way. Select the filter button at the bottom,
> 
> 1. Make sure it is selected
>    ![image](https://user-images.githubusercontent.com/19178735/203464338-c3a3ca2f-7e01-4b22-89b7-c4d12482d355.png)
> 2. Exit the software
> 3. Restart the software. At this time, the filter button at the bottom is selected. The Add Filter displayed above can be clicked at this time, or the command+F can be directly used to display the search box
>    ![image](https://user-images.githubusercontent.com/19178735/203464411-3b3fb289-ca8e-48fb-836a-2842075affbe.png)
> 
> Command+F is shown in the figure below
> 
> ![image](https://user-images.githubusercontent.com/19178735/203464464-8b7e78e3-4581-4d24-ab80-12854183ade3.png)
> 
> I'm on Mac OS Ventura (13.0) and Sequel Pro Night build 5446.

LIFE SAVER!!!
thanks!

Cant believe there more people using this app yet....

**Comment by zuernBernhard** [comment]
Sequel Ace is a very good successor of Sequel Pro - thanks to the community.

**Comment by Supparerk23** [comment]
> I found a way. Select the filter button at the bottom,
> 
> 1. Make sure it is selected
>    ![image](https://user-images.githubusercontent.com/19178735/203464338-c3a3ca2f-7e01-4b22-89b7-c4d12482d355.png)
> 2. Exit the software
> 3. Restart the software. At this time, the filter button at the bottom is selected. The Add Filter displayed above can be clicked at this time, or the command+F can be directly used to display the search box
>    ![image](https://user-images.githubusercontent.com/19178735/203464411-3b3fb289-ca8e-48fb-836a-2842075affbe.png)
> 
> Command+F is shown in the figure below
> 
> ![image](https://user-images.githubusercontent.com/19178735/203464464-8b7e78e3-4581-4d24-ab80-12854183ade3.png)
> 
> I'm on Mac OS Ventura (13.0) and Sequel Pro Night build 5446.

This solution still working!!
Thanks!

**Comment by TsaiKoga** [comment]
> > I found a way. Select the filter button at the bottom,
> > 
> > 1. Make sure it is selected
> >    ![image](https://user-images.githubusercontent.com/19178735/203464338-c3a3ca2f-7e01-4b22-89b7-c4d12482d355.png)
> > 2. Exit the software
> > 3. Restart the software. At this time, the filter button at the bottom is selected. The Add Filter displayed above can be clicked at this time, or the command+F can be directly used to display the search box
> >    ![image](https://user-images.githubusercontent.com/19178735/203464411-3b3fb289-ca8e-48fb-836a-2842075affbe.png)
> > 
> > Command+F is shown in the figure below
> > ![image](https://user-images.githubusercontent.com/19178735/203464464-8b7e78e3-4581-4d24-ab80-12854183ade3.png)
> > I'm on Mac OS Ventura (13.0) and Sequel Pro Night build 5446.
> 
> Great work!

It works, thx

**Comment by tornado-cacphy** [comment]
GOod hack. Thumbs up
